src/components/camera-tool.js

```diff
@@ -254,9 +263,35 @@ AFRAME.registerComponent("camera-tool", {
         }
         renderer.readRenderTargetPixels(this.renderTarget, 0, 0, width, height, this.snapPixels);
         pixelsToPNG(this.snapPixels, width, height).then(file => {
-          const { entity, orientation } = addMedia(file, "#interactable-media", undefined, true);
-          entity.object3D.position.copy(this.el.object3D.position).add(new THREE.Vector3(0, -0.5, 0));
+          const { entity, orientation } = addMedia(file, "#interactable-media", undefined, false);
+
+          const pos = this.el.object3D.position;
+
+          entity.object3D.position.set(pos.x, pos.y, pos.z);
           entity.object3D.rotation.copy(this.el.object3D.rotation);
+          entity.object3D.rotateY(Math.PI);
+          entity.object3D.scale.set(0.1, 0.1, 0.1);
+
+          // Generate photos in a circle around camera, starting from the bottom.
+          // Prevent z-fighting but place behind viewfinder
+          const idx = (this.localSnapCount % 6) + 3;
+
+          photoPos.set(
+            Math.cos(Math.PI * 2 * (idx / 6.0)) * 0.75,
+            Math.sin(Math.PI * 2 * (idx / 6.0)) * 0.75,
+            -0.05 + idx * 0.001
+          );
+
+          this.el.object3D.localToWorld(photoPos);
+
+          entity.setAttribute("animation__photo_pos", {
```

Comment:
We should technically have an option in media-loader for it.. but a quick hack to wait till after the image is loaded to do the aniation. Feels much nicer than animating out the cube:
```
          entity.object3D.visible = false;
          entity.addEventListener(
            "image-loaded",
            () => {
              entity.object3D.visible = true;
              entity.setAttribute("animation__photo_pos", {
                property: "position",
                dur: 800,
                from: { x: pos.x, y: pos.y, z: pos.z },
                to: { x: photoPos.x, y: photoPos.y, z: photoPos.z },
                easing: "easeOutElastic"
              });
            },
            { once: true }
          );
```